ui/src/components/settings/sidebar.rs

```diff
@@ -58,22 +60,17 @@ pub fn emit(cx: &Scope<Props>, e: Page) {
 pub fn Sidebar<'a>(cx: Scope<'a, Props<'a>>) -> Element<'a> {
```

Comment:
I didn't realize al these routes were here. in `main.rs` there's a variable called `UPLINK_ROUTES` which I used to store the `to` fields for all routes. that way there's no need to hunt for routes or remember magic strings. It may be better to use that variable here.